pkg/reconciler/v1alpha1/labeler/labeler_test.go

```diff
@@ -156,8 +157,10 @@ func routeWithTraffic(namespace, name string, traffic ...v1alpha1.TrafficTarget)
 
 func simpleRunLatest(namespace, name, config string) *v1alpha1.Route {
 	return routeWithTraffic(namespace, name, v1alpha1.TrafficTarget{
-		RevisionName: config + "-00001",
-		Percent:      100,
+		TrafficTarget: v1beta1.TrafficTarget{
+			RevisionName: config + "-00001",
```

Comment:
nit: "-00001" doesn't seem required here and we don't do it anymore so it just seems misleading.

Reply:
changed to be a random string.